api/envoy/api/v2/cds.proto

```diff
@@ -632,6 +632,25 @@ message Cluster {
   // If this flag is not set to true, Envoy will wait until the hosts fail active health
   // checking before removing it from the cluster.
   bool drain_connections_on_host_removal = 32;
+
+  // New mechanism for LB policy configuration. If set and understood
+  // by the client, replaces the lb_policy field, the common_lb_config
+  // field, and the fields in the lb_config oneof. For backward
+  // compatibility, management servers should populate both sets of fields.
+  //
+  // Every LB policy defined via this mechanism will be identified via a
+  // unique name. If the policy that needs configuration parameters, it
```

Comment:
See wording elsewhere in API for "reverse DNS" when describing the name format.

Reply:
Yeah, we could do a prefix like `io.grpc.custom.`. And we may allow LBs to register a name with their own prefix. It is quite sane to namespace the policy names here and I think there's a couple ways we can deal with it in gRPC. I'm not worried.